DEVELOPMENT.md

```diff
@@ -314,17 +314,40 @@ Read more about the colors used in Upstream in the [Color System post][cg].
 ## Proxy
 
 All of Upstream's business logic tying together the radicle code collaboration
-is provided to the UI via an HTTP API by a rust binary called the proxy. It
-uses [warp][wa] to serve a RESTish JSON API.
+is provided to the UI via an HTTP API by a rust binary called `radicle-proxy`.
+It uses [warp][wa] to serve a RESTish JSON API.
 
 For dependency management and execution of common tasks we use [Cargo][co]. To
 get up to speed with common functionality and manifest file intricacies consult
 the exhaustive [Cargo Book][cb].
 
 The proxy binary's lifecycle is managed by the main renderer of the UI in:
-`native/main.js`. When running `yarn dist` it is bundled together into an
+`native/main.ts`. When running `yarn dist` it is bundled together into an
 application package by [electron-builder][eb].
 
+### CLI options
+
+```
+$ cd proxy && cargo run --bin radicle-proxy -- --help
+
+Usage: radicle-proxy [--test]
```

Comment:
I fear that this will become outdated very quickly. We’ll probably forget to update this. Since people can just run the proxy with `--help` by themselves I’d prefer to remove this section.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1492/commits/e709a488ac28155a690c1b40068ae3da35aae615